WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -555,19 +554,19 @@ public class WMFYearInReviewDataController {
         }
     }
     
-    public func fetchUserContributionsCount(username: String, project: WMFProject?, completion: @escaping (Result<(Int, Bool), Error>) -> Void) {
+    public func fetchUserContributionsCount(username: String, project: WMFProject?, startDate: String, endDate: String, completion: @escaping (Result<(Int, Bool), Error>) -> Void) {
         guard let service = service else {
             completion(.failure(WMFDataControllerError.mediaWikiServiceUnavailable))
             return
         }
-        
+
         guard let project = project else {
             completion(.failure(WMFDataControllerError.mediaWikiServiceUnavailable))
             return
         }
         
-        let ucStartDate = "2024-11-01T22:20:13.000Z"
-        let ucEndDate = "2024-01-01T22:20:13.000Z"
+        let ucStartDate = startDate
```

Comment:
Note the API accepts these backwards - use `endDate` here and `startDate` on the next line.

Reply:
Please flip these like this, otherwise the API is not returning results for me. It's counter-intuitive (IMO) but the API expects the later date to be added to the `ucstart` parameter.

```
let ucStartDate = endDate
let ucEndDate = startDate
```